### PR TITLE
add senna.js@2.1.5 with npm auto-update

### PR DESCRIPTION
**Pull request by Amomo**
Hi @maruilian11 this PR is for #8163

I add `senna.js` at `v1.2.0` and add `npm` auto-update config in its package.json.

Would you mind checking this PR for me? Thank you~ :-D
# Profile of the lib
- Git repository (required): https://github.com/liferay/senna.js
- Official website (optional, not the repository): http://sennajs.com/
- NPM package url (optional): https://www.npmjs.com/package/senna
- GitHub / Bitbucket popularity (required):
  - Count of stars: **1426**
  - Count of watchers: 70
  - Count of forks: 131
- NPM download stats (optional):
  - Downloads in the last day: 0
  - Downloads in the last week: 40
  - Downloads in the last month: 599
# Essential checklist
- [ ] I'm the author of this library
  - [ ] I would like to add link to the page of this library on CDNJS on website / readme
- [x] This lib was not found on cdnjs repo
- [x] No already exist / duplicated issue and PR
- [x] The lib has notable popularity
  - [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
  - [x] More than 500 downloads stats per month on npm registry
- [x] Project has public repository on famous online hosting platform (or been hosted on npm)
# Auto-update checklist
- [x] Has valid tags for each versions (for git auto-update)
- [x] Auto-update setup
- [x] Auto-update target/source is valid.
- [x] Auto-update filemap is correct.
# Git commit checklist
- [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
- [x] The parent of the commit(s) in the PR is not old than 3 days.
- [x] Pull request is sending from a non-master branch with meaningful name.
- [x] Separate unrelated changes into different commits.
- [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
- [x] Close corresponding issue in commit message
- [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

File: ajax/libs/senna.js/package.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/eduardolundgren/senna.js.git"
+  },
+  "npmName": "senna",
+  "npmFileMap": [
+    {
+      "basePath": "build",
+      "files": [
+        "globals/senna*.(js+map)",
```
Comment (maruilian11) [inline ajax/libs/senna.js/package.json]: globals/senna*.+(js+map)

**Comment by maruilian11** [comment]
@Amomo please provide min file and map file if the author just give full css/js.
you can use web-minify-helper to get them

@PeterDaveHello in this case, do we need to host `senna-debug.js`? i saw the author didn't minified it, but use it for example.
http://sennajs.com/docs/

**Comment by Amomo** [comment]
@maruilian11 
I have attached the map file, and fix the typo.
Could you help me checking again? Thanks!

**Comment by maruilian11** [comment]
if you think `senna-debug.js` is necessary 
you should also provider its min file and map file.

**Comment by Amomo** [comment]
Hi @maruilian11 
After all, I remove `senna-debug.js` and `senna.js.map`.

I think that we don't need `senna.js.map` because it's a source map for knowing which files build "senna.js", not the source map for "senna.min.js".

As we host "senna.js" and "senna.min.js", but not those source files which build "senna.js".
I think we don't need `senna.js.map`.

Could you help me checking again? Thanks!

File: ajax/libs/senna.js/package.json [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/eduardolundgren/senna.js.git"
+  },
+  "npmName": "senna",
+  "npmFileMap": [
+    {
+      "basePath": "build",
+      "files": [
+        "globals/senna.+(js+map)",
```
Comment (maruilian11) [inline ajax/libs/senna.js/package.json]: this line won't grab `senna-min.js` for us.

**Comment by maruilian11** [comment]
remember assign to me after you update.

**Comment by Amomo** [comment]
@maruilian11 thank you for finding out my mistake! 😀 

I have fix the filemap of auto-update config.
Would you mind checking this PR again? Thank you!

File: ajax/libs/senna.js/package.json [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/eduardolundgren/senna.js.git"
+  },
+  "npmName": "senna",
+  "npmFileMap": [
+    {
+      "basePath": "build",
+      "files": [
+        "globals/+(senna|senna-min).js",
```
Comment (maruilian11) [inline ajax/libs/senna.js/package.json]: `.+(map|js)`

Comment (Amomo) [inline ajax/libs/senna.js/package.json]: @maruilian11 but I don't attach any .map file in "globals/" folder.

> As we host "senna.js" and "senna.min.js", but not those source files which build "senna.js".
I think we don't need senna.js.map.

Comment (maruilian11) [inline ajax/libs/senna.js/package.json]: it's for senna-min.js.map in future.

**Comment by PeterDaveHello** [comment]
Why don't you use `**/*` for build folder?

**Comment by maruilian11** [comment]
but http://sennajs.com/docs/ only mention js and css files.

**Comment by Amomo** [comment]
Hi @maruilian11 
I have update this PR to v1.4.0, could you help me checking again?
Thank you!

**Comment by maruilian11** [comment]
1.5.1 comes :sweat_smile: 
http://sennajs.com/api/ could you please ask the authors if they can also cdnjs  on website after adding this lib.

**Comment by PeterDaveHello** [comment]
Any updates here? Thanks!

**Comment by Amomo** [comment]
Hi @maruilian11 
I have updated this PR to v2.1.5.
Would you mind checking this PR again? Thank you~  😃😃😃

**Comment by sufuf3** [comment]
@Amomo Would you like to finish this PR?
Thank you.

**Comment by PeterDaveHello** [comment]
Close due to no response.

**Pull request closed without merging**